doc/nrf/gs_modifying.rst

```diff
@@ -151,8 +221,39 @@ To locate a specific configuration option, use the filter (:guilabel:`Jump to` i
 The documentation for each :ref:`configuration option <configuration_options>` also lists the menu path where the option can be found.
 
 .. important::
-   All changes to the :file:`.config` file are lost when you clean your build directory.
-   You can save it to another location, but you must then manually copy it back to your build directory.
+   All changes to the :file:`.config` file are lost when you clean your build directory or you rebuild the project with the ``pristine`` target.
+   You can save the file to another location, but you must then manually copy it back to your build directory.
+   See Zephyr's :ref:`zephyr:application_rebuild` for more information.
+
+Changing the configuration permanently
+======================================
+
+To configure your application and maintain the configuration when you clean the build directory, you need to specify the configuration in one of the permanent configuration files.
+In most cases, this will be either the default :file:`prj.conf` file of the application or one of the configuration file overlays.
+In these files, you can specify different values for configuration options that are defined by a library or board, and you can add configuration options that are specific to your application.
+
+See :ref:`zephyr:setting_configuration_values` in the Zephyr documentation for information on how to edit the :file:`prj.conf` file.
+
+.. tip::
+   When modifying configuration, especially when disabling one of the previously enabled options, check if the options required by the option you are disabling are still needed.
+
+When working with `Child images`_, you might need to permanently change their configuration.
+In such case, your application can provide overlay files for child images.
+
+If you work with SES, the :file:`prj.conf` file is read when you open a project.
+This means that after you edit this file, you must re-open your project.
+
+.. note::
+   It is possible to change the default configuration for a library by changing the :file:`Kconfig` file of the library.
+   However, best practice is to override the configuration in the application configuration file :file:`prj.conf`.
```

Comment:
We should not be mentioning that this is possible.

These two lines should be cut.

It is also possible to change the configuration by finding all uses of the option in C sources and changing it to a hardcoded value. But it's not something that we want to support or imply is a good idea.